### ISSUE TITLE
.ckpt not saving after fine-tuning run

### ISSUE THREAD

**Issue by johnfabianski** [ShivamShrirao/diffusers#93]
I run this config:
export MODEL_NAME="CompVis/stable-diffusion-v1-4"
export INSTANCE_DIR="test-training"
export CLASS_DIR="classes"
export OUTPUT_DIR="output"

accelerate launch train_dreambooth.py \
  --pretrained_model_name_or_path=$MODEL_NAME \
  --instance_data_dir=$INSTANCE_DIR \
  --class_data_dir=$CLASS_DIR \
  --output_dir=$OUTPUT_DIR \
  --instance_prompt="sks person" \
  --class_prompt="person" \
  --seed=3434554 \
  --resolution=256 \
  --center_crop \
  --train_batch_size=1 \
  --mixed_precision="fp16" \
  --use_8bit_adam \
  --gradient_accumulation_steps=1 --gradient_checkpointing \
  --learning_rate=5e-6 \
  --lr_scheduler="constant" \
  --lr_warmup_steps=0 \
  --num_class_images=100 \
  --sample_batch_size=4 \
  --max_train_steps=800

And it goes through a fine-tuning loop, but no checkpoint is saved in the output folder. Any know how to fix? Many thanks
